ext-src/NexusExplorer.ts

```diff
@@ -62,15 +62,16 @@ export class NexusExplorerProvider
   }
 
   doRefresh(): void {
-    this.reloadComponentModel().then(v => {
-      if (this.componentModel.components.length > 0) {
-        this._onDidChangeTreeData.fire();
-      }
-    });
+    this.reloadComponentModel()
```

Comment:
Got rid of `v` as we were never using it, and we return mostly empty promises to begin with (just like my parents!)